examples/live/run.sh

```diff
@@ -168,3 +168,12 @@ python -m mpi4py `which pycbc_live` \
 
 echo -e "\\n\\n>> [`date`] Checking results"
 ./check_results.py
+
+echo -e "\\n\\n>> [`date`] Running Bayestar"
+if python -c 'import sys; assert sys.version_info >= (3,6)' &> /dev/null;
```

Comment:
Right, I think this check is not necessary. Just assume Python 3.

Reply:
Yeah, I double-checked and this is the case ... I confused myself by getting a python3 error from the install and overcorrected.